NickvisionCavalier.Shared/Models/Renderer.cs

```diff
@@ -1,9 +1,21 @@
 using SkiaSharp;
+using System;
+using System.Linq;
 
 namespace NickvisionCavalier.Shared.Models;
 
 public class Renderer
 {
+    private Mirror _mirror => Configuration.Current.Mirror;
```

Comment:
just use `Configuration.Current._` directly instead of making them private properties.